Variable not in scope: document

When compiling the tests I'm getting:

```
test/DataFlow/Assertions.hs:30:49: error:                                                                                                    
    Variable not in scope: document :: Parser Diagram                                                                                           |                                                                    
30 | s `shouldReadAsDiagram` expected = shouldReadAs document s expected                                                                    
   |                                                 ^^^^^^^^                                                             
                                                                                                                                             
test/DataFlow/Assertions.hs:41:44: error:          
    Variable not in scope: document :: Parser a0                                                                                
   |                                                                                                                                        
41 | shouldFailReadAsDiagram = shouldFailReadAs document                                                                                       |                                            ^^^^^^^^
```

--- a/src/DataFlow/Reader.hs
+++ b/src/DataFlow/Reader.hs
@@ -1,18 +1,19 @@
 -- | Reads .flow syntax.
 module DataFlow.Reader (
-    readDiagram,
-    readDiagramFile
+  document,
+  readDiagram,
+  readDiagramFile
 ) where
 
-import Control.Monad
-import Data.Functor ((<$>))
-import Control.Applicative ((<*>), (<*), (*>))
-import Data.Char
-import Data.List
-import qualified Data.Map as M
-import Text.ParserCombinators.Parsec
+import           Control.Applicative           ((*>), (<*), (<*>))
+import           Control.Monad
+import           Data.Char
+import           Data.Functor                  ((<$>))
+import           Data.List
+import qualified Data.Map                      as M
+import           Text.ParserCombinators.Parsec
 
-import DataFlow.Core
+import           DataFlow.Core
 
 commentsAndSpace :: Parser ()
 commentsAndSpace = do
@@ -116,7 +117,7 @@ arrow = do
   case s of
     "->" -> return Forward
     "<-" -> return Back
-    _ -> fail "Invalid flow statement"
+    _    -> fail "Invalid flow statement"
 
 flow :: Parser Flow
 flow = do
@@ -127,7 +128,7 @@ flow = do
   i2 <- identifier
   a <- option M.empty $ try (inBraces attrs)
   case arr of
-    Back -> return $ Flow i2 i1 a
+    Back    -> return $ Flow i2 i1 a
     Forward -> return $ Flow i1 i2 a
 
 node :: Parser Node

--- a/test/DataFlow/Assertions.hs
+++ b/test/DataFlow/Assertions.hs
@@ -1,12 +1,12 @@
 module DataFlow.Assertions where
 
-import Control.Monad (when)
-import Test.Hspec
-import Text.ParserCombinators.Parsec
-import Text.Printf
+import           Control.Monad                 (when)
+import           Test.Hspec
+import           Text.ParserCombinators.Parsec
+import           Text.Printf
 
-import DataFlow.Core
-import DataFlow.Reader
+import           DataFlow.Core
+import           DataFlow.Reader               (document)
 
 parseFailure :: (Show a, Show e) => String -> a -> e -> Expectation
 parseFailure input expected err =

